src/encoded/types/quality_metric.py

```diff
@@ -330,7 +373,6 @@ class QualityMetricChipseq(QualityMetric):
     def quality_metric_summary(self, request):
         return get_chipseq_atacseq_qc_summary(self.properties, 'QualityMetricChipseq')
 
-
```

Comment:
Why deleting this line?

Reply:
I am happy to replace it. I believe my thinking was that the single space better groups the two versions of the ChIP-seq quality metric. I am a bit worried about the scenario in which someone jumps to V1 first and does not see V2.

I found spacing generally inconsistent around ChIP-seq, but I also thought a single space might be appropriate given the single line separation between QualityMetricAtacseq and QualityMetricChipseq. Maybe that should also just be two lines.